Add payment transfer test for Scalar DL

yito88 opened the pull request:
https://scalar-labs.atlassian.net/browse/DLT-3410

~It needs the latest client SDK which hasn't been released yet.~
~Unit tests should fail because the public SDK is old for now.~

~I will update the client SDK version after the latest client SDK is released.~

siyopao commented on scalardl/contract/Create.java:
```diff
@@ -0,0 +1,30 @@
+  @Override
+  public JsonObject invoke(Ledger ledger, JsonObject argument, Optional<JsonObject> property) {
+    if (!(argument.containsKey("id") && argument.containsKey("balance"))) {
+      throw new ContractContextException("required key 'id' and 'balance' is missing");
```
```suggestion
      throw new ContractContextException("required key 'id' or 'balance' is missing");
```

yito88 replied:
Thanks! Done!

siyopao commented on scalardl/contract/Transfer.java:
```diff
@@ -0,0 +1,41 @@
+    if (!(argument.containsKey("from")
+        && argument.containsKey("to")
+        && argument.containsKey("amount"))) {
+      throw new ContractContextException("required keys 'from', 'to' and 'amount' is missing");
```
```suggestion
      throw new ContractContextException("required key 'from', 'to' or 'amount' is missing");
```

yito88 replied:
Thanks! Done!

yito88 commented:
I've updated SDK and supported commit checking for unknown failure.
I found and fixed bugs the commit checking failed in my local environment.
Please review again!
